{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/templates/base.html

```diff
@@ -61,7 +61,7 @@
           {% block content_below %}{% endblock %}
         </div>
 
-        {% include 'base/_footer.html' %}
+        {% include 'footer/_footer.html' %}
```

Comment:
Yea, i also need to wrap it in a cookie cutter conditional as well. If the footer app is used then it's the added, else the `base`. Would removing it be a thing to do in the post hook?